content/en/tracing/setup_overview/setup/php.md

```diff
@@ -391,6 +391,26 @@ Enable URL's as resource names (see [Map resource names to normalized URI](#map-
 **Default**: `null`<br>
 Set an application’s version in traces and logs, for example: `1.2.3`, `6c44da20`, `2020.02.13`. Added in version `0.47.0`.
 
+`DD_TRACE_HTTP_URL_QUERY_PARAM_ALLOWED`
+: **INI**: `datadog.trace.http_url_query_param_allowed`<br>
+**Default**: `*`<br>
+A comma-separated list of query parameters to be collected as part of the URL, this can also be empty to prevent collecting any parameters or `*` to collect all parameters. Added in version `0.74.0`.
```

Comment:
```suggestion
A comma-separated list of query parameters to be collected as part of the URL. Set to empty to prevent collecting any parameters, or `*` to collect all parameters. Added in version `0.74.0`.
```